generator.rb

```diff
@@ -0,0 +1,70 @@
+def is_vowel?(letter)
+  case letter
+  when "a", "e", "i", "o", "u"
+    true
+  else
+    false
+  end
+end
+
+def run_generator
+  puts "Please enter the first word you would like to combine."
+  word1 = gets.chomp.downcase
+  until word1.length >= 2
+    puts "Please make sure the word is at least two characters."
```

Comment:
Your code to get the second word is almost exactly the same as the code to get the first word. Could you DRY this up using another method?